srcpkgs/tpm2-tss/template

```diff
@@ -1,21 +1,24 @@
 # Template file for 'tpm2-tss'
 pkgname=tpm2-tss
-version=3.0.3
+version=3.2.0
 revision=1
 build_style=gnu-configure
 configure_args="--disable-weakcrypto --with-crypto=mbed --disable-fapi"
-hostmakedepends="autoconf-archive automake libtool pkg-config doxygen libltdl-devel"
+hostmakedepends="shadow autoconf-archive automake libtool pkg-config doxygen libltdl-devel"
```

Comment:
shadow is here for adding a tss user to the chroot, I don't think we want that, is it possible to patch this out?

Reply:
Gentoo seems do to that :) Could you have a look ?